includes/class-sensei-lesson.php

```diff
@@ -3822,6 +3822,53 @@ public function get_course_id( $lesson_id ) {
 
 	}
 
+	/**
+	 * Returns the course ID for a given set of lessons (with the key being the lesson ID)
+	 *
+	 * @since $$next-version$$
+	 * @access public
+	 *
+	 * @param array<int> $lesson_ids Array of Lesson IDs.
+	 * @return array<int, int|false> Array where the key is the lesson ID, and the value is the course ID (or false if not found)
+	 */
+	public function get_course_ids( $lesson_ids ) {
+		global $wpdb;
+
+		sort( $lesson_ids, SORT_NUMERIC );
```

Comment:
Should we use `array_unique` here as well? 🤔

Reply:
Good catch! Fixed in [bc9074b](https://github.com/Automattic/sensei/pull/5294/commits/bc9074bb7a65c9e161f14b9f1799ad02562c5083).

Reply:
Just to be slightly picky...I wonder if it's better to sort first, and then run `array_unique`? I'm not sure, but I believe `array_unique` has to do a sort anyway, so I'm guessing it might be quicker if the array is already sorted? 🤷‍♂️

Reply:
Changed in [9ec32a1](https://github.com/Automattic/sensei/pull/5294/commits/9ec32a143b0faf39629852384eaa208ef94e3ddc).

I also added a `SORT_NUMERIC` parameter to `array_unique` to help with the sorting process. You were right! :)